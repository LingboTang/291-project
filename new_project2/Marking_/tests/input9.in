3
0
25
300
3
344
10
6
100
50
312
4
328
4
384
3
396
3
408
3
