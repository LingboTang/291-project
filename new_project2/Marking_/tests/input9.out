** Free memory - Before Coalesce **
Element size: 50
Pointer to next element: 268501304
Element size: 4
Pointer to next element: 268501320
Element size: 4
Pointer to next element: 268501376
Element size: 3
Pointer to next element: 268501388
Element size: 3
Pointer to next element: 268501400
Element size: 3
Pointer to next element: 0
** Busy memory - Before Coalesce **
Element size: 25
Pointer to next element: 268501292
Element size: 3
Pointer to next element: 268501336
Element size: 10
Pointer to next element: 0
** Free memory - After Coalesce **
Element size: 50
Pointer to next element: 268501304
Element size: 8
Pointer to next element: 268501376
Element size: 9
Pointer to next element: 0
** Busy memory - After Coalesce **
Element size: 25
Pointer to next element: 268501292
Element size: 3
Pointer to next element: 268501336
Element size: 10
Pointer to next element: 0

Number of chunks coalesced: 3